src/uart/registers.rs

```diff
@@ -178,3 +178,28 @@ impl<const PORT_PTR: usize> ControlRegister<PORT_PTR> {
     /// Write a 1 to flush the receive FIFO
     activate_receive_fifo_flush}
 }
+
+/// # UART Wakeup Flag Register
+/// The UART Wakeup Flag register. See Page 186, Table 12-19.
+pub struct WakeupFlagRegister<const PORT_PTR: usize> {}
+reg_impl!(RW, WakeupFlagRegister, uro::UART_WKFL);
```

Comment:
Maybe it could be helpful to change `uro` to `rro`. 

```diff
- reg_impl!(RW, WakeupFlagRegister, uro::UART_WKFL);
+ reg_impl!(RW, WakeupFlagRegister, rro::UART_WKFL);
```